products/health-care/appointments/va-online-scheduling/express-care-research/express-care-conversation-guide.md

```diff
@@ -0,0 +1,116 @@
+
+## Section 0: Warm-up questions
+
+Before we look at the website, let’s start with a few warm-up questions.
+
+- Have you heard of "Express Care"?
+  - If YES: 
+    - What is it? 
+    - Have you used it?
+  - If NO: What do you think it might be?
+  - Prompt if needed: Is it Telehealth or in-person?
+- Are you aware of any forms of Telehealth appointments offered by VA providers?
+  - If YES: 
+    - Which ones?
+  - Have you ever utilized these services? 
+    - (Prompt if needed: VA or non-VA providers)
+  - How did you become aware of them?
+
+## Section 1: Value of VA urgent Telehealth
+
+- Have you ever used Telehealth appointments?
+  - If YES: 
+    - Was this with a VA provider or non-VA doctor?
+        - If VA: How did you schedule your appointment 
+    - Was this a same-day appointment?
+    - Can you tell me a bit more about your experience?
+  - If NO: 
+    - In your own words, can you define "Telehealth"?
+    - Have you considered it before? 
+    - What would convince you to try it?
+  - Would you consider using Telehealth appointments for a same-day appointment?
+- Have you heard anything about same-day/urgent care Telehealth appointments through VA?
+  - (Clarification question if needed: Do you think that VA offers any same-day urgent care appointments? Do you think that the VA offers any same-day urgent care Telehealth appointments?)
+- Have you ever tried to make a same-day appointment with a VA provider, either someone you see regularly or any available provider?
+  - If YES: 
+    - Can you tell me what happened?
+    - How did you make this appointment?
+    - Were you able to be seen?
+  - If NO:
+    - Have you ever considered doing this, but something prevented you? If so, what?
+
+## Section 2: Scheduling expectations
+
+- Do you consider "same-day" or urgent care the same thing as scheduling an appointment? If not, how is this different?
+- Can you tell me what types of urgent care you might seek through VA's same-day Telehealth appointments?
+- Who do you expect to speak with for an urgent same-day VA Telehealth appointment?
+
+## Section 3: Coronavirus appointments 
+
+Next, I'm going to ask you some questions related to coronavirus. These are general questions related to how you might seek medical help if you are experiencing symptoms. If you feel uncomfortable at any point we can move on to the next section or stop, just let me know. 
+
+- When you are speaking with family or friends how do you refer to the current pandemic?
+- Have you been practicing physical distancing, physical isolation, or quarantined, by choice or otherwise?
+- Where do you go to seek guidance on taking care of yourself and your family during this pandemic?
+  - What VA health care resources are you aware of related to coronavirus, if any?
+- Have you had any concern that you have experienced symptoms related to coronavirus?
+  - If NO: Imagine that you did, what would be your first 
```

Comment:
What would be your first symptom? or first step?

Reply:
whoops! clearly got distracted here!

Reply:
@lenaecb updated with a full thought

Reply:
Have you had any concern that you have experienced symptoms related to coronavirus?
- If YES:
     - What did you do to get care?
- If NO:
     - Imagine that you did, what would your first step towards seeking care be?
- Are you aware of any coronavirus screening resources through VA available to you?